index.bs

```diff
@@ -1499,7 +1499,7 @@ a variable <var>stream</var>, that performs the following steps:
     </ol>
   <li> Set <var>stream</var>@\[[writing]] to <b>true</b>.
   <li> Let <var>writeResult</var> be PromiseInvokeOrNoop(<var>stream</var>@\[[underlyingSink]], <code>"write"</code>,
-    «‍<var>writeRecord</var>.\[[chunk]]»).
+    « <var>writeRecord</var>.\[[chunk]]»).
```

Comment:
Oh, I see. I copied your U+200D while replaced it with SP at writeRecord. I wasn't aware that. Anyway, thanks for fixing and committing.